.nfpm_template.yaml

```
@@ -0,0 +1,20 @@
+# nfpm configuration file used as template.
+# all "ig_" will be replaced by proper values in the release pipeline.
+name: "ig"
+arch: "ig_arch"
+platform: "linux"
+version: "ig_version"
+section: "devel"
+replaces:
+- ig
+provides:
+- ig
+maintainer: "Francis Laniel <flaniel@linux.microsoft.com>"
+description: |
+  Introspecting and debugging Kubernetes applications using eBPF "gadgets"
```

```suggestion
  Introspecting and debugging Linux containers using eBPF "gadgets"
```

However I'd ask the help of @blixtra to write it.

---

I'd go with my suggestion in the time being as this tool is not only about Kubernetes.